src/graceful/resources/base.py

```diff
@@ -241,20 +241,32 @@ def require_params(self, req):
                 try:
                     if param.many:
                         # params with "many" enabled need special care
-                        params[name] = param.container(
-                            req.get_param_as_list(name, param.value) or
-                            [param.default and param.value(param.default)]
-                        )
+                        values = req.get_param_as_list(
+                            # note: falcon allows to pass value handler using
+                            #       `transform` param so we do not need to
+                            #       iterate through list manually
+                            name, param.validated_value
+                        ) or [
+                            param.default and
+                            param.validated_value(param.default)
```

Comment:
Hmm, I think we need to reconsider the whole idea of using default values for parameters as "raw". I don't remember why it is so. I think this is something that was left in time when we could not do it differently. Now it seams that we could completely get rig to that limitation.

Let's make it another issue and research later.